lib/inject.js

```diff
@@ -117,7 +117,7 @@ function injectAllRequirements(funcArtifact) {
   } else {
     return injectRequirements(
       path.join('.serverless', 'requirements'),
-      this.serverless.service.package.artifact,
+      this.serverless.service.package.artifact || funcArtifact
```

Comment:
Getting a syntax error. I think you're missing the `,` at end of line.

Reply:
Ah. yes! (and that's probably why depcheck didn't think we use glob-all too)